api/v1alpha1/constants.go

```diff
@@ -53,5 +53,6 @@ const (
 	HAServiceAnnotationsKey            = "skipnodeport.ako.vmware.com/enabled"
 	HAAVIInfraSettingAnnotationsKey    = "aviinfrasetting.ako.vmware.com/name"
 
-	AKODeploymentConfigControllerName = "akodeploymentconfig-controller"
+	AKODeploymentConfigControllerName                                = "akodeploymentconfig-controller"
+	ClusterIpFamilyValidationFailedCondition clusterv1.ConditionType = "ClusterIpFamilyValidationFailed"
```

Comment:
1. it is better to group it with other condition, put it into L48
2. The condition description should be something like `AKOIPFamilyValidtionFailed` instead of cluster